lin-alg/V9.sage

```diff
@@ -5,47 +5,41 @@ class V9(MastrExercise):
 
   def generate(self):
 	
+        vec=[]
 	#Pick 6 vectors in R4
-	vec1 = vector([ randrange(-10,10), randrange(-10,10), randrange(-10,10), randrange(-10,10)])
-	vec2 = vector([ randrange(-10,10), randrange(-10,10), randrange(-10,10), randrange(-10,10)])
-	vec3 = vector([ randrange(-10,10), randrange(-10,10), randrange(-10,10), randrange(-10,10)])
-	vec4 = vector([ randrange(-10,10), randrange(-10,10), randrange(-10,10), randrange(-10,10)])
-	vec5 = vector([ randrange(-10,10), randrange(-10,10), randrange(-10,10), randrange(-10,10)])
-	vec6 = vector([ randrange(-10,10), randrange(-10,10), randrange(-10,10), randrange(-10,10)])
+        for _ in range(0,5):
+		vec.append( vector([ randrange(-6,6), randrange(-6,6), randrange(-6,6), randrange(-6,6)]) )
```

Comment:
This was on my todo list, sorry about the hacky first go :) . The problem with using yet another language is I don't have all the stupid syntax memorized. Several others will need to be similarly refactored.